src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class Figure implements CalculateArea, Drawable {
```

Comment:
remove redundant empty lines, replace setters with a constructor, remove random and figType fields and abstract methods

Reply:
I Can't  replace setters with a constructor for area couse i calculeite it